curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-34-digit-factorials.english.md

```diff
@@ -22,7 +22,7 @@ Note: as 1! = 1 and 2! = 2 are not sums they are not included.
 ```yml
 tests:
   - text: '<code>digitFactorial()</code> should return { sum: 40730, numbers: [145, 40585] }.'
```

Comment:
```suggestion
  - text: <code>digitFactorial()</code> should return { sum: 40730, numbers: [145, 40585] }.
```

Reply:
@moT01 Your suggested change shows removing a space before the `- text:` which would result in invalid yaml.  I don't mind removing the surrounding double quotes though.  I will just take care of removing them in a separate commit.